libtransact/src/scheduler/multi/mod.rs

```diff
@@ -73,7 +74,7 @@ pub struct MultiScheduler {
 impl MultiScheduler {
     /// Returns a newly created `MultiScheduler` that runs the specified sub-schedulers.
     pub fn new(
-        mut schedulers: Vec<Box<dyn Scheduler + Send>>,
```

Comment:
This looks like it belongs in the Send commit

Reply:
Good catch; fixed